data/wof-schema.sql

```diff
@@ -11,3 +12,13 @@ CREATE TABLE wof_neighbourhood (
 CREATE INDEX wof_neighbourhood_wof_id_index ON wof_neighbourhood(wof_id);
 CREATE INDEX wof_neighbourhood_label_position_index ON wof_neighbourhood USING GIST(label_position);
 CREATE INDEX wof_neighbourhood_geometry_index ON wof_neighbourhood USING GIST(geometry);
+
+CREATE TABLE wof_neighbourhood_placetype (
+  placetype_code SMALLINT UNIQUE NOT NULL,
```

Comment:
Do we also want to make `wof_neighbourhood`'s `placetype SMALLINT NOT NULL REFERENCES wof_neighbourhood_placetype(placetype_code)` so that Postgres will check the FK relationship for us? It's of marginal benefit, given the small size of the table, but might catch an error later down the line.